auth.c

```diff
@@ -991,6 +991,11 @@ auth_restrict_session(struct ssh *ssh)
 
 	/* A blank sshauthopt defaults to permitting nothing */
 	restricted = sshauthopt_new();
+#ifdef WINDOWS
```

Comment:
This can be an upstream change as well. 

Create an upstream bug like this https://bugzilla.mindrot.org/show_bug.cgi?id=3403